#Message for autonomy to make transit start

bool stop
float32 x
float32 y
